tests/topology.py

```diff
@@ -91,38 +123,43 @@ def test_out_of_bounds(self):
         topology.resize(4)
 
         residue = Residue("")
-        residue.add_atom(1)
-        topology.add_residue(residue)
+        residue.atoms.append(1)
+        topology.residues.append(residue)
+
+        _ = topology.atoms[2]
```

Comment:
I don't understand the meaning of the `_` here. Mind explaining it?

Reply:
`_ = whatever()` is a way to ignore the returned value, while still making it clear that the function is returning something.

Reply:
Thanks!